Overview.bs

```diff
@@ -102,8 +102,17 @@ There are two different methods which can be used to incrementally transfer font
 Patch Subset {#patch-subset}
 ----------------------------
 
-The first method,
-<a href="patch-incxfer">Patch Subset</a>, uses a server which can generate binary updates to an existing font file.
+In the the first method, <a href="patch-incxfer">Patch Subset</a> a server generates binary patches which a
+client applies to a subset of the font in order to extend the coverage of that subset. The server
+is stateless, it does not maintain any session data for clients between requests. Thus when a client
+requests the generation of a patch from the server it has to fully describe the current subset of the
+font that it has in a way which allows the server to recreate it.
+
+Generic binary patch algorithms are used which do not need to be aware of the specifics of the font
+format. Typically a server will produce a patch by generating two font subsets: one which matches what
+the client currently has and one which matches the extended subset the client desires. A binary patch
+is then produced between the two subsets.
+
 
```

Comment:
I'd like to avoid making statements in the spec that might constitute a judgement call. 
For example, stating that "Range Request is a very simple method of incremental transfer ..." is both misleading and a judgement call at the same time. I'd argue that while "Patch Subset" requires a more complex server-side support, it is simpler to implement on the UA side that already support WOFF2, Also, with Patch Subset a UA doesn't need to parse font data to figure out how to request and retrieve the next incremental update - one just asks for additional characters to be included. 

The Range Request method doesn't have server-side requirements and can be deployed on most of the servers without much effort, which is an advantage for some use cases, but implementations are definitely not going to be "very simple" - parsing partially downloaded font data to figure out glyph coverage and decide what byte range(s) needs to be requested (especially if complex scripts / layout tables are concerned) isn't something I'd call simple.

I'd suggest to make the overview a bit more descriptive to discuss why two different methods are offered and what's involved in the process of choosing one over another. In the end, it will be authors' decision which one to use, the UAs will need to be able to support both.

Reply:
I agree that making one part simpler has the trade off that other parts become more complex.